tests/backup/backup_helper.go

```diff
@@ -58,6 +62,11 @@ const (
 	mongodbPodStatusRetryTime                 = 30 * time.Second
 	backupLocationDeleteTimeout               = 30 * time.Minute
 	backupLocationDeleteRetryTime             = 30 * time.Second
+	latestPxBackupVersion                     = "2.4.0"
+	latestPxBackupHelmBranch                  = "master"
+	pxCentralPostInstallHookJobName           = "pxcentral-post-install-hook"
```

Comment:
post_install_hook_pod is already declared in portworx.go  as post_install_hook_pod ,although it does not follow camel convention.
Either correct that to camel convention and reuse here or vice-versa